app/jobs/distribution_reminder_job.rb

```diff
@@ -1,9 +1,11 @@
 # This job notifies a Partner that they have a distribution scheduled to be sent in 24 hours
-class DistributionReminderJob
-  include Sidekiq::Worker
-
+class DistributionReminderJob < ApplicationJob
```

Comment:
Feels like this doesn't have to be a `Job` at all? The async stuff happens in `DistributionMailer`? If so, then we can probably just make this into a simple class?

Reply:
Possibly, but that's out of scope for this PR. I'll make an issue to explore that :+1:

Reply:
#1616